server-spi-private/src/main/java/org/keycloak/policy/BlacklistPasswordPolicyProviderFactory.java

```diff
@@ -39,28 +39,33 @@
  * A blacklist-file is referred to by its name in the policy configuration.
+ *
+ * <h1>Blacklist location</h1>
  * <p>Users can provide custom blacklists by adding a blacklist password file to the configured blacklist folder.
  * <p>
  * <p>The location of the password-blacklists folder is derived as follows</p>
  * <ol>
  * <li>the value of the System property {@code keycloak.password.blacklists.path} if configured - fails if folder is missing</li>
  * <li>the value of the SPI config property: {@code blacklistsPath} when explicitly configured - fails if folder is missing</li>
- * <li>otherwise {@code ${jboss.server.data.dir}/password-blacklists/} if nothing else is configured - the folder is created automatically if not present</li>
+ * <li>otherwise {@code $KC_HOME/data/password-blacklists/} if nothing else is configured</li>
  * </ol>
- * <p>Note that the preferred way for configuration is to copy the password file to the {@code ${jboss.server.data.dir}/password-blacklists/} folder</p>
- * <p>To configure a password blacklist via the SPI configuration, run the following jboss-cli script:</p>
- * <pre>{@code
- * /subsystem=keycloak-server/spi=password-policy:add()
- * /subsystem=keycloak-server/spi=password-policy/provider=passwordBlacklist:add(enabled=true)
- * /subsystem=keycloak-server/spi=password-policy/provider=passwordBlacklist:write-attribute(name=properties.blacklistsPath, value=/data/keycloak/blacklists/)
- * }</pre>
- * <p>A password blacklist with the filename {@code 10_million_password_list_top_1000000-password-blacklist.txt}
- * that is located beneath {@code /data/keycloak/blacklists/} can be referred to
- * as {@code 10_million_password_list_top_1000000-password-blacklist.txt} in the <em>Authentication: Password Policy</em> configuration.
+ *
+ * To configure the blacklist folder via CLI use {@code --spi-password-policy-password-blacklist-blacklists-path=/path/to/blacklistsFolder}
+ *
+ * <p>Note that the preferred way for configuration is to copy the password file to the {@code $KC_HOME/data/password-blacklists/} folder</p>
+ * <p>A password blacklist with the filename {@code 10_million_passwords.txt}
+ * that is located beneath {@code $KC_HOME/data/keycloak/blacklists/} can be referred to as {@code 10_million_passwords.txt} in the <em>Authentication: Password Policy</em> configuration.
+ *
+ * <h1>False positives</h1>
+ * <p>
+ * The current implementation uses a probabilistic data-structure called {@link BloomFilter} which allows for fast and memory efficient containment checks, e.g. whether a given password is contained in a blacklist,
+ * with the possibility for false positives. By default a false positive probability {@link #DEFAULT_FALSE_POSITIVE_PROBABILITY} is used.
+ *
+ * To change the false positive probability via CLI configuration use {@code --spi-password-policy-password-blacklist-false-positive-probability=0.00001}
+ * </p>
```

Comment:
What about adding this as new bullets to the documentation of the [Password blacklist policy](https://github.com/keycloak/keycloak-documentation/blob/main/server_admin/topics/authentication/password-policies.adoc)?

Reply:
I just sent a PR with the changes for the keycloak documentation: https://github.com/keycloak/keycloak-documentation/pull/1742/files